controller.rb

```diff
@@ -0,0 +1,37 @@
+require 'pry'
+require_relative 'Parser'
+require_relative 'deck'
+require_relative 'view'
+
+class Controller
+  # include Parser
+  attr_reader :viewer, :deck
+
+  def initialize(file)
+    @parsed_file = Parser.read(file)
+    @viewer = View.new
+    @deck = Deck.new(@parsed_file)
+    @deck.generate_random_card
+    @viewer.welcome_message
+    run_flashcard_game
+  end
+
+  def run_flashcard_game
+    @viewer.show_definition(@deck.flashcards.first)
```

Comment:
Rather than the Controller knowing that the Deck has an array of flashcards which has a `first` method, I would write a `#current_card` method on the Deck, and then it's the Deck's job to know what that means.